src/main/groovy/jenkins/automation/builders/BaseJobBuilder.groovy

```diff
@@ -25,46 +26,18 @@ class BaseJobBuilder {
     String description
     List<String> emails
 
-    Job build(DslFactory factory){
-        factory.job(name){
+    Job build(DslFactory factory) {
+        factory.job(name) {
             it.description this.description
-            addBaseStuff(delegate,this.emails)
-        }
-    }
-
-/**
- *  Adds colorized input plugin support
- * @param context
- */
-    static void addColorizeOutput(context){
-
-        context.with{
-            colorizeOutput()
-        }
-    }
-
-/**
- *  Adds a minimum base functionality required -build claiming, notifications and log.
- * @param context  delegate passed in context
- * @param emails  email list of the developers for notification
- */
-    static void addBaseStuff(context, emails) {
-        context.with{
-            wrappers{
-                addColorizeOutput(delegate)
-            }
-            logRotator {
-                //TODO: make it parametrized to builder
-                numToKeep(10)
-            }
-            publishers{
-                allowBrokenBuildClaiming()
-
-                if(emails){
+            CommonUtils.add_defaults(delegate)
```

Comment:
we can leave it with a sensible default and optional ability to override. Net total effect is the same with configure block, but they achieve it with a one-liner parameter.